manifests/04-config.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: openshift-console
+  namespace: openshift-console-operator
```

Comment:
Lots of changing the same values across the manifests.  I want something like `helm template` to be able to use `consts` across these, but hopefully wont have tons of changes in the future.